### DISCUSSION TITLE
追夢格林無限期禁賽

### DISCUSSION THREAD

**Discussion by 12Jesse24** [1101-GS4719-C/Python-Community-GS4719-C#1783] [General]
時間發生在2023/12/13，勇士作客太陽。他在第三節一次場邊發球的卡位時竟然對著太陽中鋒（Jusuf Nurkic)頭部揮拳，當下就吞下二級惡意犯規直接遭驅逐出場，根據紀錄這是他本賽季第三次被驅逐出場。且依照聯盟的規則，可以按照過去的紀錄進行加罰。我原本以為會禁個10-20場而已，結果這次竟然是無限期禁賽，看了真是大快人心，每次看到他在場上搞事，真的很不爽，就不能好好控制自己的情緒嗎。各位覺得這動作需要被禁賽幾場呢?
https://www.youtube.com/watch?v=DZ7jh0b1G5M

**Comment by Tomken123** [comment]
嘴綠這人就是這樣，真性情，對我來說它是一個真男人，敢打敢衝狠狠

**Comment by uwje** [comment]
格林被禁賽不簽來看對球隊好壞參半，以結果來說，湯普森的表現有所回穩，但在跟金塊隊的比賽中因為沒有他，導致倒不住對面的球員，所以希望他能盡快調整好心態回歸球場，找回自己的身手

**Comment by Terryyyyyyyyyyyy** [comment]
我是支持他繼續在場上的，畢竟他也是蠻有用的，不管是在組織還是擋拆都還蠻有用的，雖然禁賽後鐵湯反而更優秀，但是留在場上還是對球隊有幫助的，而且在打不過的時候只要執行1換1，或許還有獲勝的機會

**Comment by alvin119** [comment]
不管他在場上打球多麼有用，打人就是有問題(而且他還是故意的)，
除了敗壞自己的名聲，也敗壞了其他隊友甚至是整個聯盟的名聲，
看在過去的經歷無限期禁賽很合情合理

**Comment by B1nn9** [comment]
雖然長期來說他在勇士體系中算是不可或缺的存在，但他打球真的太髒了，而且也絲毫沒有要改進的意思，這種球員對於場上另一方的球員無疑是顆不定時炸彈，所以無限期禁賽也合情合理，至少要禁到讓他主動做出實質改變為止。
